setup/default.xml

```diff
@@ -24,6 +24,7 @@
     <entry key='logger.level'>info</entry>
     <entry key='logger.file'>./logs/tracker-server.log</entry>
     <entry key='logger.rotate'>true</entry>
+    <entry key='logger.rotate.interval'>day</entry>
```

Comment:
Day is the default indeed. I thought about not including it, but then I came to a conclusion, that it won't do any harm, instead this will make the configuration more obvious and straightforward.

I will remove it, if you insist

Reply:
We try to avoid including all possible options in there. Otherwise it would become too big and it would be hard to find something people really need often, like file paths etc.

Reply:
removed